plugins/versionpress/src/Git/CommitMessage.php

```diff
@@ -57,6 +57,19 @@ public function getSubject()
     }
 
     /**
+     * Returns commit message subject without prefix set in constant VP_COMMIT_MESSAGE_PREFIX
+     * 
+     * @return string
+     */
+    public function getUnprefixedSubject()
```

Comment:
Guys I don't agree that `GitRepository` is the right place for handling how the message should be presented to user. It should just obtain commit messages from underlaying GIT repository as is without manipulating it. 

If I strip it right in the `GitRepository` now, we could get rid of the prefix at all, because there will be no benefits of having it in commit messages. In proposed implementation, we will need to do some work to obtain the prefix, because it will be stripped by default. I understand that stripping can be optional but it will add some noise there.

For me letting `CommitMessage` class provide option to return it's properties in filtered or unfiltered way is better than noising `GitRepository`. For me is this approach a little bit similar to Java's `.toString()` methods.